weasyl/character.py

```diff
@@ -227,8 +227,10 @@ def _select_character_and_check(userid, charid, rating=None, ignore=True, anyway
     elif ignore and blocktag.check(userid, charid=charid):
         raise WeasylError('TagBlocked')
 
+    query = dict(query.items())
```

Comment:
I had read somewhere that you needed to do `.items()`, suppose I should've tested that for myself. 

Anyway, updated now.